cmd/create/cluster/cmd.go

```diff
@@ -1789,7 +1799,15 @@ func run(cmd *cobra.Command, _ []string) {
 	computeNodes := args.computeNodes
 	// Compute node requirements for multi-AZ clusters are higher
 	if multiAZ && !autoscaling && !isReplicasSet {
-		computeNodes = 3
+		if isHostedCP {
```

Comment:
the behavior for this is the same as in the other changes, right?
Could it be some function that computes amount of min replicas?

Reply:
+1 to this, if we could break it to a function and add a unit test for this functionality :pray: